vru_use_case_scripts/plot_detected_objects

```diff
@@ -159,6 +159,12 @@ parser.add_argument(
     required=True,
 )
 
+parser.add_argument(
+    "--plots-dir",
+    type=Path,
+    help="Directory to store generated plots",
```

Comment:
maybe just make a default value instead of plots dir and figures dir which would also add documentation by indicating in the help the default value.